applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -378,6 +381,11 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 
 	LOG_INF("Connected: %s", addr);
 	default_conn = bt_conn_ref(conn);
+
+	ret = ble_mcs_discover(default_conn);
```

Comment:
Hmmm, looks like if we do MCS discover from here then controller will be very busy at the beginning.
Have you seen the stability of ACL/ISO link changed?
If something changed, then we might need to add a security update callback handler, and do MCS discover when securitfy update is finished.

Reply:
Didn't see any change in stability

Reply:
I think this is a good point. It would be better to not stress the controller more than needed. It does not look like it requires much rewrite to move ble_mcs_discover()?

Reply:
@alexsven

Reply:
Moved it to lc3_cap_config_cb